http/src/main/scala/SSE.scala

```diff
@@ -57,19 +57,18 @@ object SSE {
           .map(writeTo(sink))
           .run.run
 
+  /*
+  There was a bug where despite a "halt" command, the process was not terminating, leaking resources.
+
+  Now when the client disconnects, sink.write will throw an IOException. This exception will bring the Process down.
+   */
   private def writeTo(sink: java.io.Writer): String => Process[Task, Unit] =
     line => Process.eval(Task {
       sink.write(line)
       sink.flush // this is a line-oriented protocol,
                  // so we flush after each line, otherwise
                  // consumer may get delayed messages
-    }.attempt).flatMap(_.fold(e => e match {
-      case x: java.io.IOException =>
-      // when client disconnects we'll get a broken pipe
-      // IOException from the above `sink.write`. This
-      // gets translated to normal termination
-      Process.halt
```

Comment:
Better change is to replace this line with 

    Process.halt.kill

This will not throw uncatched exception and process will die nicely (better with `.halt` but `halt` is buggy)